src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    Calculator calculator = new Calculator();
+
+    @BeforeAll
+    static void calculatorImplementsInterface_True() {
+        Calculator calculator = new Calculator();
+        assertTrue(calculator instanceof Calculable,
+                "Must inherit interface behavior");
+    }
```

Comment:
This test has a sense in your implementation of the calculator. But I don't see, why do you declare two variables of the same name and assign them Calculator object. I would expect something like that

```suggestion
    Calculator calculator;

    @BeforeAll
    static void calculatorImplementsInterface_True() {
        calculator = new Calculator();
        assertTrue(calculator instanceof Calculable,
                "Must inherit interface behavior");
    }
```

Reply:
Wrong. I can't access non-static field from a static context.

Reply:
sure, but what if you put at line 11 static? how would that change the whole test behavior?

Reply:
You can do this field static, and then all will be ok

Reply:
My bad about creating that redundant variable in the first place. Removed.

Reply:
We are here to learn, mistakes are inevitable)